src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,54 @@
+package study;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static final String DEFAULT_DELIMITER = ",|:";
+    public static final String REGEX = "//(.)\n(.*)";
+
+    public static int splitAndSum(String text) {
+        if (StringUtils.isBlank(text)) {
+            return 0;
+        }
+
+        String[] tokens = splitText(text);
+
+        int sum = 0;
+        for (String token : tokens) {
+            sum = sum + toPositiveInt(token);
+        }
+
+        return sum;
+    }
+
+    public static String[] splitText(String text) {
+        Matcher m = Pattern.compile(REGEX).matcher(text);
+        if (m.find()) {
+            String customSeparator = m.group(1);
+            text = m.group(2);
```

Comment:
```suggestion
            String customSeparator = m.group(1);
            text = m.group(2);
```
`1`,`2`같은 매직넘버를 의미있는 이름으로 표현해보아도 좋을것 같네요!